### PR TITLE
support better per-cluster naming

### PR DESCRIPTION
**Pull request by dhellmann**
This set of changes is the first in a series to support running
dev-scripts multiple times on the same host to create multiple
clusters. With these changes in place, the files created end up either
in cluster-specific locations or with the cluster name in the file
name. It is also possible to control the names of the provisioning and
baremetal networks using variables in the config file and control the
starting value for the VMBC ports used by the emulator.

### CODE REVIEW

**Comment by metal3ci** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/1544/

**Comment by stbenjam** [comment]
The changes look reasonable to me - could you also update run_ci.sh?

**Comment by metal3ci** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/1545/

File: 02_configure_host.sh [hunk 1]
```diff
@@ -149,30 +152,29 @@ for PORT in 80 ${LOCAL_REGISTRY_PORT} ; do
...
         fi
     fi
 done
 
 # Allow ipmi to the virtual bmc processes that we just started
-VBMC_MAX_PORT=$((6230 + ${NUM_MASTERS} + ${NUM_WORKERS} - 1))
 if [ "${RHEL8}" = "True" ] ; then
-    sudo firewall-cmd --zone=libvirt --add-port=6230-${VBMC_MAX_PORT}/udp
-    sudo firewall-cmd --zone=libvirt --add-port=6230-${VBMC_MAX_PORT}/udp --permanent
+    sudo firewall-cmd --zone=libvirt --add-port=${VMBC_BASE_PORT}-${VBMC_MAX_PORT}/udp
```
Comment (dhellmann) [inline 02_configure_host.sh]: This should be VBMC.

**Comment by metal3ci** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/1546/

**Comment by dhellmann** [comment]
I've extended this now to include the default names of the networks based on the cluster name, to run a separate httpd instance on the provisioning host for each network, and to be more explicit about some other values in install-config.yaml.

**Comment by metal3ci** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/1548/

File: ocp_install_env.sh [hunk 2]
```diff
@@ -107,9 +107,14 @@ controlPlane:
     baremetal: {}
 platform:
   baremetal:
+    libvirtURI: qemu+ssh://${PROVISIONING_HOST_USER}@${PROVISIONING_HOST_IP}/system
```
Comment (stbenjam) [inline ocp_install_env.sh]: Was this intentional to include? Shouldn't it be optional since the default is `qemu:///system`?

Comment (stbenjam) [reply ocp_install_env.sh]: I assume that's why the latest dev-scripts run failed

Comment (dhellmann) [reply ocp_install_env.sh]: I need it there for hive, and I was trying to just be consistent and always include it. I'll make it configurable and deal with that in a separate PR.

**Comment by metal3ci** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/1552/

**Pull request merged**